compile-angel.el

```diff
@@ -660,41 +810,35 @@ existence.
 The function iterates through the extensions in `load-file-rep-suffixes` to
 construct possible .el file paths. If a matching file exists, return its path;
 otherwise, return nil."
-  (cond
-   ((not file)
-    (compile-angel--debug-message
-     "compile-angel--find-el-file: nil file")
-    nil)
-
-   ((compile-angel--is-el-file file)
-    file)
-
-   ((string-equal (file-name-extension file) "elc")
-    (let ((base (file-name-sans-extension file))
-          (suffixes load-file-rep-suffixes)
-          result)
-      (while (and suffixes (not result))
-        (let ((candidate (concat base ".el")))
-          (when (file-exists-p candidate)
-            (setq result candidate)))
-        (setq suffixes (cdr suffixes)))
-      result))
-
-   (t
-    (compile-angel--debug-message
-     "compile-angel--find-el-file: NO .el FILE CORRESPONDS TO: %s" file))))
+  (compile-angel--with-fast-file-ops
+    (cond
+     ((not file) nil)
```

Comment:
Please add back:
```
((not file)
    (compile-angel--debug-message
     "compile-angel--find-el-file: nil file")
    nil)
```

It can be useful for debugging purposes.

Reply:
Okay. I have re-added it back in.